ndb/src/google/cloud/ndb/model.py

```diff
@@ -1472,23 +1492,6 @@ def _serialize(
         """
         raise NotImplementedError
 
-    def _deserialize(self, entity, p, unused_depth=1):
```

Comment:
ISTM it'll be easier to track "completion" of the implementation by leaving methods like this as `raise NotImplementedError` and then marking them for deletion at a later time

Reply:
We're already using NotImplementedError to signal API we haven't gotten to yet. How do you propose distinguishing between API we haven't gotten to yet and API we don't intend to implement?

Reply:
Add a message to the `NotImplementedError`, as is done in e.g. in [`Key.from_old_key()`][1]

[1]: https://github.com/googleapis/google-cloud-python/blob/b69461c2e6c856783ca5b9b3eaafdc56d8fcc6db/ndb/src/google/cloud/ndb/key.py#L778